docs/running-e2e-tests.md

```diff
@@ -0,0 +1,41 @@
+## Running Kubernetes e2e tests
+
+This document explains how to run Kubernetes e2e tests with rktlet.
+
+First, start the rktlet, optionally with a more verbose output:
+
+```
+$ sudo ./bin/rktlet -v=4
+```
+
+On another terminal, go to the Kubernetes source directory and start a local cluster.
+We set some flags because tests use those features (`ALLOW_*`) and we configure the cluster to use the CRI, and rktlet in particular.
+
+```
+$ cd $GOPATH/src/k8s.io/kubernetes
+$ sudo ALLOW_PRIVILEGED=true \
+       ALLOW_SECURITY_CONTEXT=true \
+       CONTAINER_RUNTIME=remote \
+       CONTAINER_RUNTIME_ENDPOINT=/var/run/rktlet.sock \
+       CGROUP_DRIVER=systemd \
+       ./hack/local-up-cluster.sh
+
+[...]
```

Comment:
You probably want to end the block with three backticks and start a new one below the "To start using you cluster, …".

Reply:
This is actually `local-up-cluster.sh`'s output. I agree it's a bit confusing.

Reply:
Maybe I'll just remove that part and only leave the command invocation